recipes/liboptv/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: optv
+  version: {{ version }} 
+
+source:
+  url: https://github.com/OpenPTV/openptv/archive/{{ version }}.tar.gz
+  sha256: 4cafc70d966f2694cc936973eb2581d35bdffd6ec4b30f9ff99a1515c07f4042
+
+build:
+  number: 0
+  skip: True  # [not py3k]
+  script:
+  - cd py_bind
+  - {{ PYTHON }} setup.py prepare
+  - {{ PYTHON }} setup.py install
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy 1.16.1
```

It's probably safer not to specify a minimum patch version
```suggestion
    - numpy 1.16
```